cli/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gluster/glusterd2/cli/cmd"
+)
+
+func main() {
+	// Migrate old format Args into new Format. Modifies os.Args()
```

Comment:
Is there a way to get the log initialized here before we do anything else?

Right now, it's being initialized during package initialization of `cmd`, which technically happens before this point, but it doesn't allow for much configurability.